source/js/components/join/join.scss

```diff
@@ -17,16 +17,6 @@
       width: map-get($glyph-container, "width");
       margin: auto;
     }
-
-    .privacy-error {
```

Comment:
These rules aren't doing anything because of`tw-ml-2` set in `join.jsx` around line 553 overrides them. Removing.